index.html

```diff
@@ -193,11 +193,19 @@ <h3>npm</h3>
           <p><code>npm install -g purescript</code></p>
           <p><small>(Installation via <code>npm</code> requires Node version 8 or later)</small></p>
 
+          <h3>Homebrew</h3>
+          <p><code>brew install purescript</code><p>
+
+          <h3>Nix</h3>
+          <p><code>nix-env -i purescript</code></p>
+          <p><small>(Install using Nix v2.3+ and a recent version of <code>&lt;nixpkgs&gt;</code>)</small></p>
+
           <h3>Tools</h3>
-          <p>The recommended build tool is Spago, which can be installed using <code>npm</code>:</p>
+          <p>The recommended build tool for Purescript is <a href="https://github.com/purescript/spago/blob/master/README.md" title="Spago: Purescrpt's package manager and build tool">Spago</a>, which can be installed using <code>npm</code>:</p>
           <p><code>npm install -g spago</code></p>
           <p>You might like to install some of these
             <a href="https://github.com/purescript/documentation/blob/master/ecosystem/Editor-and-tool-support.md">additional tools and editor plugins</a>.</p>
+          <p>NPM or Yarn is recommended for managing Javascript dependencies in your project.</p>
```

Comment:
Note that recommending Yarn might prompt users to install purescript locally via yarn, which does not currently work due to PnP compatibility.

Reply:
I will amend, thanks.